### DISCUSSION TITLE
Meeting types and topics

### DISCUSSION THREAD

**Discussion by lzachmann** [lzachmann/models-for-missing-data#15] [General]
Hey everybody,

I'm thinking about two basic types of meetings going forward, both of which I think have a lot of value. First, are "office hours" &ndash; typically 1:1 meetings that aren't on any sort of regular schedule. We've had a lot of these calls this week! The other type &ndash; "focal topic team calls" &ndash; is the real reason for this post. These could be standing (recurrent) calls involving more people and focused on specific topics of general interest. If we're organized enough, we could populate a calendar with topic ideas just so we know what's on the agenda. If you all like this idea, feel free to give this a 👍 , and perhaps suggest a topic or two. I figured we might shoot to get on a more consistent schedule after the New Year if that works? Here's a couple of ideas just to get the ball rolling.

- [ ] Using Git and navigating around your file system using Git Bash
  - Find some sort of 'Intro to Git' tutorial on YouTube. We don't want anything complicated. The basic 'verbs' we need to familiarize ourselves with include clone, status, add, commit, pull, push, and checkout. It may also be valuable to know how to fix merge conflicts when they occur. Another quick video on using a shell (Git Bash) to run Git commands and get around your file system may also be super helpful. The end result is that we'll all be in sync with updates as the tool, data, and model configs continue to evolve.
- [ ] Refresher on model parameterizations, and model outputs, the workflow
  - E.g., `b0` vs `b0-b1` or fixed stratum vs hierarchical site variances, how do we choose. Do I use the Poisson or negative binomial? When is a zero-inflated model needed. Where are all of the outputs I need to look at and how do we evaluate a given model, and across models?
- [ ] How do I use some of our model outputs to produce my own custom plots (for reports / presentations, etc)?
- [ ] My model converges and passes posterior predictive checks, but how do I know how 'good' it is?
  - Introducing observed vs predicted plots, MAE or RMSE, etc.

**Comment by megcswan** [comment]
Other suggestions:
Choosing and using covariates (esp. using terrain covariates properly)
Specifying and interpreting marginal effects
Developing our own simple models

**Reply by lzachmann** [reply]
These are _great_ suggestions, thanks Megan.

**Reply by megcswan** [reply]
One more: deep dive into model outputs. Especially model checking and diagnostics folders. We know there is a lot of great content in there, but many of us aren't sure what it tells us! We can also use this discussion to flesh out the SOP.

**Comment by megcswan** [comment]
Data exploration: Things you should do with your data prior to running a model
